### PR TITLE
Removing the note about aria-owns not being supported in Safari on iOS and macOS

### LINKED ISSUE #33436: Note about VoiceOver is not valid anymore
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-owns

### What specific section or headline is this issue about?

Description

### What information was incorrect, unhelpful, or incomplete?

The last note about VoiceOver not having support for aria-owns, links to a page that shows that all test passes.

### What did you expect to see?

No note...

### Do you have any supporting links, references, or citations?

https://a11ysupport.io/tech/aria/aria-owns_attribute

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/accessibility/aria/attributes/aria-owns`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-owns
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/attributes/aria-owns/index.md
* Last commit: https://github.com/mdn/content/commit/019ca5c9ce641bfa02825e1ba0444f35dfb646cc
* Document last modified: 2024-04-17T05:19:10.000Z

</details>

### PR DESCRIPTION
**Pull request by PeterSaathoff-Harshfield**
Removing the note about aria-owns not being supported in Safari on iOS and macOS because it is now supported. See https://github.com/accessibilitysupported/a11ysupport.io/issues/281 for more info.

### Description

Removing a note about aria-owns not being supported on iOS and macOS because it is now supported. I tested with iOS and macOS VoiceOver using multiple web pages.

### Motivation

This note is out of date.

### Additional details

https://github.com/accessibilitysupported/a11ysupport.io/issues/281 
https://mastodon.social/@pesh/112045384811778674

### Related issues and pull requests

Fix https://github.com/mdn/content/issues/33436

### CODE REVIEW

File: files/en-us/web/accessibility/aria/attributes/aria-owns/index.md [hunk 1]
```diff
@@ -42,8 +42,6 @@ The {{CSSXRef('order')}} property, part of flex or grid layouts, can be used to
 
 Make sure your owned elements have only one owner. Do not specify the `id` of an element in more than one other element's `aria-owns` attribute. An element can have only one owner.
 
-> **Warning:** At the time of this writing, [`aria-owns` is not supported](https://a11ysupport.io/tech/aria/aria-owns_attribute) on MacOS and iOS with VoiceOver.
```
Comment (scottaohara) [inline files/en-us/web/accessibility/aria/attributes/aria-owns/index.md:45]: i don't think it's a good idea to remove the warning all together, as that doesn't acknowledge the fact there would still be an "issue" for people who were using / need to support versions of Safari prior to the release this change went out in.

Comment (estelle) [reply files/en-us/web/accessibility/aria/attributes/aria-owns/index.md:45]: How about changing this to the following when a11ysupport.io confirms safari support:

```suggestion
> **Warning:** While [`aria-owns` is now supported](https://a11ysupport.io/tech/aria/aria-owns_attribute) in all modern browsers, `aria-owns` is not exposed to users of MacOS and iOS using VoiceOver prior to version X.
```
where X is the version listed on https://a11ysupport.io/tech/aria/aria-owns_attribute

**Comment by bsmth** [comment]
Hi @PeterSaathoff-Harshfield - thanks a lot for the suggestions. There's some review comments outstanding for you to take a look at. Do you plan to come back to this one? Thank you! :)

File: files/en-us/web/accessibility/aria/attributes/aria-owns/index.md (hunk 1 above)
Comment (PeterSaathoff-Harshfield) [reply files/en-us/web/accessibility/aria/attributes/aria-owns/index.md:47]: I like this suggestion. a11ysupport.io now shows iOS/macOS being supported, but I don't see any version numbers referenced there, and I also didn't notice exactly which version made the change. All I know is, it was somewhere between iOS 16.0/macOS 13.0 and iOS 17.3/macOS 14.3.

**Comment by PeterSaathoff-Harshfield** [comment]
Sorry, I'm very unfamiliar with how to do this and don't want to make a mistake. I commented above because the suggestion by estelle included a version number but we unfortunately don't know the exact version it changed.

File: files/en-us/web/accessibility/aria/attributes/aria-owns/index.md (hunk 1 above)
Comment (scottaohara) [reply files/en-us/web/accessibility/aria/attributes/aria-owns/index.md:47]: support was added around feb 2024 (right before this issue was filed / a11ysupport was updated)

i'm sure a quick google search will help determine what version number was released around that time.

File: files/en-us/web/accessibility/aria/attributes/aria-owns/index.md [hunk 2]
```diff
@@ -44,8 +44,6 @@ The {{CSSXRef('order')}} property, part of flex or grid layouts, can be used to
...
```
Comment (estelle) [inline files/en-us/web/accessibility/aria/attributes/aria-owns/index.md:47]: How about:

```suggestion
> **Warning:** While [`aria-owns` is now supported](https://a11ysupport.io/tech/aria/aria-owns_attribute) in all modern browsers, `aria-owns` may not be exposed to users of MacOS and iOS using VoiceOver prior to iOS 17.3 and macOS 14.3.
```

by adding "may" we don't need to be exact, but are enabling this to be edited out in the future when both of those versions are almost obsolete.

Comment (estelle) [reply files/en-us/web/accessibility/aria/attributes/aria-owns/index.md:47]: @PeterSaathoff-Harshfield Let me know you're ok with this suggestion by merging. Or, feel free to edit.

File: files/en-us/web/accessibility/aria/attributes/aria-owns/index.md [hunk 3]
```diff
@@ -44,8 +44,6 @@ The {{CSSXRef('order')}} property, part of flex or grid layouts, can be used to
 
 Make sure your owned elements have only one owner. Do not specify the `id` of an element in more than one other element's `aria-owns` attribute. An element can have only one owner.
```
Comment (estelle) [inline files/en-us/web/accessibility/aria/attributes/aria-owns/index.md:45]:
```suggestion
Make sure your owned elements have only one owner. Do not specify the `id` of an element in more than one other element's `aria-owns` attribute. An element can have only one owner.

> **Warning:** While [`aria-owns` is now supported](https://a11ysupport.io/tech/aria/aria-owns_attribute) in all modern browsers, `aria-owns` may not be exposed to users of MacOS and iOS using VoiceOver prior to iOS 17.3 and macOS 14.3.
```

**Review by estelle — APPROVED** [review]
Thank you! 🎉

**Pull request merged**